src/styles/blocks/card.scss

```
@@ -0,0 +1,107 @@
+.catalog__wrapper {
+  grid-area: catalog;
+  display: grid;
+  grid-gap: 46px 48px;
+  padding-bottom: 72px;
+  padding-block: 50px;
+  padding-inline: 40px;
+  @include columns;
+}
+
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px; /* 150% */
+    margin: 0 0 4px;
+
+    &:hover {
+      transition: color 300ms;
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    color: $secondary-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px; /* 140% */
+    margin-bottom: 16px;
+  }
+
+  &__wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &--reviews {
+      margin-bottom: 24px;
+    }
+
+    &--price {
+      margin-bottom: 16px;
+    }
+  }
+
+  &__review {
+    color: $main-accent;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__price-key {
+    color: $secondary-color;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px; /* 150% */
```

Remove comments everywhere
```suggestion
    line-height: 18px;
```